internal/pkg/cli/config/config.go

```diff
@@ -195,6 +195,7 @@ func flags() *pflag.FlagSet {
 	f.String("ahead-block-size", cfg.AheadBlockSize.String(), "Size of a one read ahead input block.")
 
 	f.String("input-size-threshold", cfg.InputSizeThreshold.String(), "At least one slice must exceed the threshold, otherwise the table is copied without modification.")
+	f.Uint32("input-size-low-exit-code", cfg.InputSizeLowExitCode, "If specified, the skipped tables is not be copied, but the program exits with the exit code.")
```

Comment:
You are right, standard approach is that exit codes are defined in documentation.

I thought differently during the implementation:
- It is a one-time tool that is called from PHP.
- I preferred explicit definition via flag:
  - PHP programmer see, in his code, the flag with explicitly defined exit code.
  - And a few lines below he will have a check the actual exit code returned by the process.
  - So from the PHP code it is clear how it works.
- Our colleagues from PS team asked us to make it adjustable in detail (so they don't have to bother us with it), that's why we have 25 flags.
- I didn't find any other CLI that would handle it this way.
- **I have no problem adjusting it as you suggest, but it seems like a waste of time for me.**.
  - Because real impact of the change will be IMHO zero.
  - I'm not stopping you from improving it yourself if you want.

Reply:
Yes.